.github/workflows/branch_sync.yml

```diff
@@ -54,74 +56,47 @@ jobs:
       - name: Configure git
         uses: cylc/release-actions/configure-git@v1
 
-      - name: Attempt fast-forward
-        id: ff
+      - name: Checkout sync branch if it exists
         continue-on-error: true
         run: |
-          git merge "origin/${HEAD_BRANCH}" --ff-only
-          git push origin master
+          git switch -c "$SYNC_BRANCH" "origin/${SYNC_BRANCH}"
+          echo "BASE_BRANCH=$SYNC_BRANCH" >> "$GITHUB_ENV"
 
-      - name: Check for existing PR
```

Comment:
Shouldn't be a need to check for the PR explicitly as the existence of the `8.*.x-sync` branch almost guarantees a PR is open